doc/rtd/conf.py

```diff
@@ -28,13 +28,14 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = [
+    'm2r',
     'sphinx.ext.autodoc',
     'sphinx.ext.autosectionlabel',
     'sphinx.ext.viewcode',
 ]
 
 # The suffix of source filenames.
-source_suffix = '.rst'
+source_suffix = ['.rst', '.md']
```

Comment:
As we're explicitly including `SECURITY.md` by path in `security.rst`, do we need to expand this list?

Reply:
It appears we do not. I was following the steps on the m2r site when I added this, but it looks like the import handles this for us.

Reply:
reverted in latest push